### PR TITLE
RzTable: replace UT64MAX with dashes for X specifier

### PR DESCRIPTION
**Pull request by p371jah**
**Your checklist for this pull request**
- [ ] I've read the [guidelines for contributing](https://github.com/rizinorg/rizin/blob/master/DEVELOPERS.md) to this repository
- [ ] I made sure to follow the project's [coding style](https://github.com/rizinorg/rizin/blob/master/DEVELOPERS.md#code-style)
- [ ] I've added tests that prove my fix is effective or that my feature works (if possible)
- [ ] I've updated the documentation and the [rizin book](https://github.com/rizinorg/book) with the relevant information (if needed)

**Detailed description**

...

**Test plan**

...

**Closing issues**

...

### CODE REVIEW

File: librz/core/cbin.c [hunk 1]
```diff
@@ -2276,7 +2276,7 @@ static int bin_symbols(RzCore *r, PJ *pj, int mode, ut64 laddr, int va, ut64 at,
 		}
 	}
 	if (IS_MODE_NORMAL(mode)) {
-		rz_table_set_columnsf(table, "dssssdss", "nth", "paddr", "vaddr", "bind", "type", "size", "lib", "name");
+		rz_table_set_columnsf(table, "dXXssdss", "nth", "paddr", "vaddr", "bind", "type", "size", "lib", "name");
```
Comment (ret2libc) [inline librz/core/cbin.c:2279]: so before paddr and vaddr were considered as strings??

**Review by ret2libc — COMMENTED** [review]
LGTM, though I will download locally the changes and check some tests because GitHub web ui is not able to show large diffs :(

File: librz/core/cbin.c (hunk 1 above)
Comment (p371jah) [reply librz/core/cbin.c:2279]: Yep, these fields were converted directly with `sdb_fmt()`.

File: librz/core/cbin.c [hunk 2]
```diff
@@ -2450,10 +2450,10 @@ static int bin_symbols(RzCore *r, PJ *pj, int mode, ut64 laddr, int va, ut64 at,
 			const char *type = symbol->type ? symbol->type : "NONE";
 			const char *name = rz_str_get(sn.demname ? sn.demname : sn.name);
 			// const char *fwd = rz_str_get (symbol->forwarder);
-			rz_table_add_rowf(table, "dssssdss",
+			rz_table_add_rowf(table, "dXXssdss",
 				symbol->ordinal,
-				symbol->paddr == UT64_MAX ? " ----------" : sdb_fmt(" 0x%08" PFMT64x, symbol->paddr),
```
Comment (p371jah) [inline librz/core/cbin.c:2455]: @ret2libc This extra whitespace is the reason of many changes in tests.

**Review by XVilka — APPROVED** [review]
Looks certainly better now. Good catch!

**Review by ret2libc — APPROVED** [review]

**Comment by ret2libc** [comment]
Thanks!

**Review by Maijin — APPROVED** [review]

**Pull request merged**